### ISSUE TITLE
Add support for EPSM/YMF288 audio expansion

### ISSUE THREAD

**Issue by MinecraftMario27** [TASEmulators/fceux#680]
https://famistudio.org/doc/expansion/

EPSM is not an audio expansion that ever used, or even existed in the lifetime of the NES, but rather the pet project of Perkka who designed a small circuit board that can be plugged in the expansion port of the NES and controlled by it to produce sound. The expansion is based off the Yamaha YMF288 chip and adds 3 extra square channels with shared noise capabilities, 6 FM synthesis channels and 6 rhythm channels. The square are fixed at a duty cycle or 50%. Those are essentially the same as Sunsoft S5B. The FM channels are 4-OP channels that can be configured independently. The rhythm channels are 6 pre-defined drum samples.
FamiStudio has EPSM/YMF288 audio expansion. FamiStudio has NES/FDS ROM export. NES/FDS ROM export supports EPSM/YMF288.

**Comment by negativeExponent** [comment]
any usecase for this other than playing music? anything playable at the moment.

**Comment by MinecraftMario27** [comment]
> any usecase for this other than playing music? anything playable at the moment.

Yes, FamiStudio has EPSM YMF288 sound chip. FamiStudio is NES/Famicom 8-bit music maker similar to FamiTracker. FamiStudio has ROM Export (.nes, .fds, .unf, .unif) and NES Sound Format NSF (.nsf)

Do we have plan to add EPSM/YMF288 support. EPSM (Expansion Port Sound Module) YMF288 has 3 Square channels same as Sunsoft 5B sound chip, which is AY-3-8910/YM2149 clone, 6 FM channel synth and 6 Percussions/Drums.
